pkg/streaming/commands/stream_create_test.go

```diff
@@ -133,9 +133,6 @@ spec:
   provider: test-provider
 status:
   address: {}
-  binding:
```

Comment:
this is still relevant as it existed in system independent of the service name enhancement

Reply:
Huh! I didn't _intend_ to delete that. Reinstated.